pyartifactory/models/repository.py

```diff
@@ -306,3 +312,16 @@ class RemoteRepositoryResponse(RemoteRepository):
     enableVagrantSupport: bool = False
     enableGitLfsSupport: bool = False
     enableDistRepoSupport: bool = False
+
+
+class FederatedRepository(LocalRepository):
+    """Models a federated Repository."""
+
+    rclass: Literal[RClassEnum.federated] = RClassEnum.federated
+    members: List[FederatedMembers] = []
+
+
+class FederatedRepositoryResponse(FederatedRepository):
+    """Models a federated repository response."""
+
+    members: List[FederatedMembers] = []

```

Comment:
No need of that if you're inheriting from `FederatedRepository`. I wonder if there are other hidden params (not documented on the API) returned. Can you please share the result of a FederatedRepositoryResponse using the API ?

Reply:
```
FederatedRepository(key='test2'
 projectKey=None
 rclass=<RClassEnum.federated: 'federated'>
 packageType=<PackageTypeEnum.generic: 'generic'>
 description=None
 notes=None
 includesPattern='**/*'
 excludesPattern=''
 repoLayoutRef='maven-2-default'
 checksumPolicyType=<ChecksumPolicyType.client_checksums: 'client-checksums'>
 handleReleases=True
 handleSnapshots=True
 maxUniqueSnapshots=0
 maxUniqueTags=0
 debianTrivialLayout=False
 snapshotVersionBehavior=<SnapshotVersionBehavior.non_unique: 'non-unique'>
 suppressPomConsistencyChecks=False
 blackedOut=False
 xrayIndex=False
 propertySets=None
 dockerApiVersion='V2'
 archiveBrowsingEnabled=False
 calculateYumMetadata=False
 yumRootDepth=0
 enableFileListsIndexing=False
 optionalIndexCompressionFormats=None
 downloadRedirect=False
 cdnRedirect=False
 blockPushingSchema1=False
 primaryKeyPairRef=None
 secondaryKeyPairRef=None
 priorityResolution=False
 cargoInternalIndex=False
 members=[FederatedMembers(url='https://testdomain.de/artifactory/test2'
 enabled=True)])
```